portfolio/src/main/webapp/script.js

```diff
@@ -26,4 +26,20 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
-//console.log('hiii');
\ No newline at end of file
+//console.log('hiii');
+
+function getMessage(){
+    const fetchPromise = fetch('/data');
+    fetchPromise.then(handleResponse);
+}
+
+function handleResponse(response){
+    const responsePromise = response.text();
+    responsePromise.then(createMessage);
+}
+
+function createMessage(messageText){
+    console.log(messageText);
```

Comment:
I'd also remove this debugging statement too. This can be seen by any public user who inspects the console. We can always add it back when we are debugging.